tools/config.py

```diff
@@ -464,6 +465,17 @@ def add_config_files(self, flist):
             self.lib_config_data[cfg["name"]] = cfg
 
     @property
+    def building_bootloader(self):
+        """Does this config have regions defined?"""
```

Comment:
Please change this text to describe the property. It currently is the same as `has_regions` and does not return the same thing.